src/client/routes/signup/index.js

```diff
@@ -9,6 +9,7 @@ import Success from '../../components/info/success';
 import { signUp } from '../../api/authentication';
 import { setToken } from '../../helpers/token';
 import config from '../../config';
+import { userLogin } from '../../actions';
```

Comment:
This import somehow did not go through in my last PR, sorry about that. That is why I have added it here.